06-process.Rmd

```diff
@@ -64,17 +64,17 @@ The below are the important prerequisites of a successful data visualization pro
++ **Understanding the audience** How will they process this visualization is another important prerequisite. Designing a visualization for scientists is entirely different from a visualization designed for law-makers or for general public. Most user want to see 'Key performance indicators' which are the main drivers for visualizations. The following are different kinds of visualizations that can be used to display KPIs:
+    + Quantities: counts or measures. Example - Count of likes or comments.  
+    + Trends and changes over time:  time series. Example - Change in sales quantity over time.  
+    + Relatives Share and proportions:  display relationship between the parts and the whole. Example - breakdown of a stock portfolio by asset.  
+    + Ranked list: although not a real data visualization, it could achieve the goal needed.  
+    + Geographical Location: Gives user spatial and physical relationships.  
 
- **Understanding the data you are trying to visualize** such as shape, dimension of the data, is data a time-series, relationship in data between entities,categorical attributes is also an important prerequisite. Is this data from a single source or multiple sources?. If the data is from multiple sources, they need to be blended together without loosing the meta data constraints and business rules. When the data is collected, the end goal should be kept in mind.
++ **Understanding the data you are trying to visualize** such as shape, dimension of the data, is data a time-series, relationship in data between entities,categorical attributes is also an important prerequisite. Is this data from a single source or multiple sources?. If the data is from multiple sources, they need to be blended together without loosing the meta data constraints and business rules. When the data is collected, the end goal should be kept in mind.
 
 + **Deciding which visual is best**
-One of the biggest challenges for business users is deciding which visual should be used to best represent the information. Tables, Linechart, Areachart, Barchart, Scatterplots, Piechart, Treemaps, Heatmaps, Natual Language Generation, Etc.
+One of the biggest challenges for business users is deciding which visual should be used to best represent the information. Tables, Linechart, Areachart, Barchart, Scatterplots, Piechart, Treemaps, Heatmaps, Natual Language Generation, Etc. Depending upon the industry and specific department business may want to measure success based on specific goals and targets. Few of the KPI examples that can help are listed as follows: Marketing Metrics, Sales Metrics, Saas Metrics and KPIs, Digital Marketing Metrics, Social Media Metrics, SEO KPIs, Financial Metrics and KPIs, Supply Chain Metrics, Healthcare Metrics, Call Center Metrics, Support Metrics, Retail Metrics and KPIs, Help Desk Metrics and KPIs, Insurance Metrics, E-Commerce Metrics and KPIs, HR Metrics, Devops Metrics and KPIs. 
```

Comment:
Great addition, but could you list out the metrics in a bullet point format? It would help readability tremendously.

Reply:
Hi @harizhisham , Agreed! That's a good suggestion. I included the bullet formatting now.